test/integration/admin_login_test.rb

```diff
@@ -33,6 +34,16 @@ def setup
     assert page.has_content?(category1.name)
   end
 
+  test "an admin user has a unique email" do
+    @user1 = User.create(username: 'userd',
+                        password: 'password',
+                        first_name: 'Johnn',
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.